build.gradle

```diff
@@ -1,6 +1,10 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.5.32"
+    ext {
+        kotlin_version = '1.5.21'
+        compose_version = '1.0.1'
```

Comment:
Not the latest version, but I was unable to get a more recent version working with any Kotlin version.

Reply:
Just curious, is there any reason not to use a more recent version of kotlin too? As a candidate I might find it struggling to not be able to use recent APIs.

Reply:
Does this help? https://developer.android.com/jetpack/androidx/releases/compose-kotlin#pre-release_kotlin_compatibility

Reply:
BTW as of very recently, the Compose compiler version (which is what matters for Kotlin compatibility) is decoupled from the Compose library version

Reply:
Can I just go ahead and update to the latest Kotlin & Compose compiler versions?

Reply:
Yeah why not